packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/worker.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import { resolve } from 'path';
+import { Configuration } from 'webpack';
+import { WebpackConfigOptions } from '../build-options';
+import { getTypescriptWorkerPlugin } from './typescript';
+
+const WorkerPlugin = require('worker-plugin');
+
+
+export function getWorkerConfig(wco: WebpackConfigOptions): Configuration {
+  const { buildOptions } = wco;
```

Comment:
Maybe put the `const WorkerPlugin = require('worker-plugin');` in this method to require it only when needed.

Reply:
I agree with the perf argument but it goes against the general pattern we use for imports. I'd prefer to not go against the grain without a good reason, and this plugin is very small so I don't think there is much to be gained.